dart/test/sentry_client_test.dart

```diff
@@ -102,6 +102,28 @@ void main() {
       expect(capturedEvent.exceptions?.first.stackTrace, isNotNull);
     });
 
+    test('should attach isolate info in thread', () async {
```

Comment:
I'll need to change this test because it should only be executed on dart:io platforms. Web stays as it was before this PR, because it doesn't has isolates.